cmd/shell.go

```diff
@@ -209,42 +209,49 @@ func FailOnError(err error, msg string) {
 func AmqpChannel(conf Config) (*amqp.Channel, error) {
 	var conn *amqp.Connection
 
-	if conf.AMQP.SSL != nil {
-		if strings.HasPrefix(conf.AMQP.Server, "amqps") == false {
-			err := fmt.Errorf("SSL configuration provided, but not using an AMQPS URL")
+	if conf.AMQP.TLS == nil {
+		// Configuration did not specify SSL options
+		conn, err := amqp.Dial(conf.AMQP.Server)
+		if err != nil {
 			return nil, err
 		}
 
-		cfg := new(tls.Config)
+		return conn.Channel()
+	}
 
-		cert, err := tls.LoadX509KeyPair(conf.AMQP.SSL.CertFile, conf.AMQP.SSL.KeyFile)
-		if err != nil {
-			err = fmt.Errorf("Could not load Client Certificates: %s", err)
-			return nil, err
-		}
-		cfg.Certificates = append(cfg.Certificates, cert)
+	// They provided SSL options, so let's load them.
+	blog.GetAuditLogger().Info("Loading SSL Options.")
+
+	if strings.HasPrefix(conf.AMQP.Server, "amqps") == false {
```

Comment:
I don't see a need to make such a drastic change to the development and testing environments in this PR. We can open another ticket if that's a concern.

As long as I can configure the Let's Encrypt environment to use TLS with mutual auth, that's all that I care about. It'll get documented into the [Implementation Guide](https://github.com/letsencrypt/boulder/wiki/Implementation-Guide) for other people wanting to deploy Boulder as a CA.